otter/worker/_rcv3.py

```diff
@@ -27,6 +28,11 @@ def _generic_rcv3_request(step_class, request_bag, lb_id, server_id):
         request has no body.
     """
     effect = step_class(lb_node_pairs=[(lb_id, server_id)])._bare_effect()
+
+    svc_req = effect.intent
```

Comment:
Yep. Alas, 409 is really wide, so allowing 409 means also allowing a bunch of other failures. What do you think the appropriate way forward is?